src/traits/window.rs

```diff
@@ -6,8 +6,21 @@ use glib::translate::{from_glib_none, ToGlibPtr};
 use ffi;
 use glib::to_gboolean;
 use cast::GTK_WINDOW;
+use gdk;
 
 pub trait WindowTrait : ::WidgetTrait {
+    fn move_(&self, x: i32, y: i32) {
```

Comment:
[isize](http://doc.rust-lang.org/std/primitive.isize.html). That page is somewhat out of date and confusing because it talks about Rust's `int` type which is different from C `int`.

`usize`/`isize` (former `uint`/`int`) are guaranteed to fit a pointer while C's `int` is on all current platforms equivalent to `i32`.